<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUqerAccessDialog</class>
 <widget class="QDialog" name="CUqerAccessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CUqerAccessDialog</string>
  </property>
  <widget class="QLineEdit" name="m_editToken">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4516631ff8298c847d332eb457a6fba0aa8caf6690dd640d92ce609718ab3ba1</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_labelToken">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Token:</string>
   </property>
   <property name="buddy">
    <cstring>m_editToken</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="m_labelUrl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>URL:</string>
   </property>
   <property name="buddy">
    <cstring>m_editURL</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_editURL">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_editReply">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>651</width>
     <height>321</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnGetData">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>GetData</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>m_btnGetData</sender>
   <signal>clicked()</signal>
   <receiver>CUqerAccessDialog</receiver>
   <slot>slot_BtnGetDate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_BtnGetDate()</slot>
 </slots>
</ui>
